test/cli.spec.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
```

Comment:
Please do not disable linting rules on top of the file.. Respect the coding standard defined in the project